plugin/src/main/java/com/denizenscript/denizen/events/entity/AreaEffectCloudApplyScriptEvent.java

```diff
@@ -0,0 +1,51 @@
+package com.denizenscript.denizen.events.entity;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.AreaEffectCloudApplyEvent;
+
+public class AreaEffectCloudApplyScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // area cloud applies effects
+    //
+    // @Group Entity
+    //
+    // @Cancellable true
+    //
+    // @Triggers when a lingering potion's area cloud applies its effect to an entity, and every five ticks after.

```

Comment:
This is a bit unclear - it's not that the event fires when effects are applied and keeps firing every 5 ticks, the effects are actually applied every 5 ticks from what I can understand (or presumably it's based on the effect cloud's reapplication delay, which defaults to 5 ticks? can test in-game and see).

Also can just say it's when an area effect cloud applies it's effects, lingering potions are just 1 feature that creates effect clouds, they can exist in other ways.